backend/rollcall/tasks.py

```diff
@@ -76,9 +71,8 @@ def _send_user_timesheet(user):
 '''
     message = EmailMessage('SahaBee User Timesheet',
                            body_text,
-                           from_email=user.detail.work_email,
-                           to=[receiver_email],
-                           cc=cc_list,
+                           from_email='sahabee@mymail.sahab.ir',
```

Comment:
No need to specify it explicitly.  
Please remove the line at all or, if you insist, use `settings.DEFAULT_FROM_EMAIL`.